modules/core/client/views/sidebar.client.view.html

```diff
@@ -0,0 +1,40 @@
+<section id="sidebar">
+    <br>
```

Comment:
!fix Using br to move stuff down is bad practice, use a custom margin class to push down content if you must. !fix